docs/src/main/sphinx/security/opa-access-control.md

```diff
@@ -212,6 +218,99 @@ The `targetResource` is used in cases where a new resource, distinct from the on
   }
 }
 ```
+## RowFilters 
+Row filtering allows Trino to filter out some rows from the result before
+returning it to the caller, controlling what data different users can see. This
+plugin supports retrieving filters from OPA.
+
+For these policies to be enabled, make sure to set `opa.policy.row-filters-uri`
+in the OPA plugin configuration.
+
+For instance, an OPA policy for row filtering may be implemented as below:
+
+```rego
+  package trino
+  import future.keywords.in
+  import future.keywords.if
+  import future.keywords.contains
+
+  default allow := true
+
+  table_resource := input.action.resource.table
+  is_admin {
+    input.context.identity.user == "admin"
+  }
+
+  rowFilters contains {"expression": "user_type <> 'customer'"} if {
+      not is_admin
+      table_resource.catalogName == "sample_catalog"
+      table_resource.schemaName == "sample_schema"
+      table_resource.tableName == "restricted_table"
+  }
+```
+The response expected by the plugin is an array of objects, each of them in the
+format `{"expression":"clause"}`.
+
+Each expression essentially behaves like an additional `WHERE` clause.
```

Comment:
Detail some more what that means with the rego script in mind .. what does it do?

Reply:
To be done later